std/http/_io.ts

```diff
@@ -346,7 +346,7 @@ export async function readRequest(
   req.conn = conn;
   req.r = bufr;
   [req.method, req.url, req.proto] = firstLine.split(" ", 3);
-  [req.protoMinor, req.protoMajor] = parseHTTPVersion(req.proto);
+  [req.protoMajor, req.protoMinor] = parseHTTPVersion(req.proto);
```

Comment:
Yikes, that's a pretty bad typo, @Noxazer-FR can you add some test case ensuring we don't regress?

Reply:
Yes of course, don't hesitate to leave comment on my work if necessary.